helion/_compiler/host_function.py

```diff
@@ -246,6 +249,44 @@ def codegen_function_def(self, statements: list[ast.AST]) -> ast.FunctionDef:
             type_params=None,
         )
 
+    def codegen_call_function(self) -> ast.FunctionDef:
+        def stringify(arg: object) -> str:
+            if isinstance(arg, (list, tuple)):
+                rhs = [stringify(a) for a in arg]
+            rhs = arg
+            if isinstance(arg, torch.SymInt):
+                rhs = int(arg)
+            elif isinstance(arg, torch.SymFloat):
+                rhs = float(arg)
+            elif isinstance(arg, torch.SymBool):
+                rhs = bool(arg)
```

Comment:
Do we need to be using size hints here?  `SymInt.__int__` adds a guard implicitly which might create some weirdness in the shape env?

Reply:
I think there is a helper to get the size hint in CompileEnvironment.